src/main/java/org/javarosa/core/model/instance/CsvExternalInstance.java

```diff
@@ -27,6 +28,7 @@ public static TreeElement parse(String instanceId, String path) throws IOExcepti
                 }
 
                 root.addChild(item);
+                multiplicity++;
```

Comment:
No test for this one?

Reply:
The test in that commit does verify it. With the rename to `itemsFromExternalSecondaryCSVInstance_ShouldBeAvailableToXPathParser` as you suggested, that should hopefully be clearer now.